boards/arm/mimxrt1064_evk/doc/mimxrt1064_evk.rst

```diff
@@ -0,0 +1,198 @@
+
+Connections and I/Os
+====================
+
+The MIMXRT1064 SoC has five pairs of pinmux/gpio controllers.
+
++---------------+-----------------+---------------------------+
+| Name          | Function        | Usage                     |
++===============+=================+===========================+
+| GPIO_AD_B0_09 | GPIO            | LED                       |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_12 | LPUART1_TX      | UART Console              |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_13 | LPUART1_RX      | UART Console              |
++---------------+-----------------+---------------------------+
+| WAKEUP        | GPIO            | SW0                       |
++---------------+-----------------+---------------------------+
+
+System Clock
+============
+
+The MIMXRT1064 SoC is configured to use the 24 MHz external oscillator on the
+board with the on-chip PLL to generate a 600 MHz core clock.
+
+Serial Port
+===========
+
+The MIMXRT1064 SoC has eight UARTs. One is configured for the console and the
+remaining are not used.
+
+Programming and Debugging
+*************************
+
+The MIMXRT1064-EVK includes the :ref:`nxp_opensda` serial and debug adapter
+built into the board to provide debugging, flash programming, and serial
+communication over USB.
+
+To use the Segger J-Link tools with OpenSDA, follow the instructions in the
+:ref:`nxp_opensda_jlink` page using the `Segger J-Link OpenSDA V2.1 Firmware`_.
+The Segger J-Link tools are the default for this board, therefore it is not
+necessary to set ``OPENSDA_FW=jlink`` explicitly when you invoke ``make
+debug``.
+
+With these mechanisms, applications for the ``mimxrt1064_evk`` board
+configuration can be built and debugged in the usual way (see
+:ref:`build_an_application` and :ref:`application_run` for more details).
+
+The pyOCD tools do not yet support this SoC.
+
+Flashing
+========
+
+The Segger J-Link firmware does not support command line flashing, therefore
+the usual ``flash`` build system target is not supported.
+Instead, see the NXP `How to Enable Boot from QSPI Flash App Note
+<https://www.nxp.com/docs/en/application-note/AN12108.pdf>`_ for flashing instructions.
+
+
+Debugging
+=========
+
+This example uses the :ref:`hello_world` sample with the
+:ref:`nxp_opensda_jlink` tools. Run the following to build your Zephyr
+application, invoke the J-Link GDB server, attach a GDB client, and program
+your Zephyr application to flash. It will leave you at a GDB prompt.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: mimxrt1064_evk
+   :goals: debug
+
+
+.. _MIMXRT1064-EVK Website:
+   https://www.nxp.com/support/developer-resources/run-time-software/i.mx-developer-resources/mimxrt1064-evk-i.mx-rt1064-evaluation-kit:MIMXRT1064-EVK
+
+.. _MIMXRT1064-EVK User Guide:
+   https://www.nxp.com/docs/en/user-guide/PMSMFOCRT1050UG.pdf
```

Comment:
This doesn't seem to be a user guide for the EVK but for something else

Reply:
thanks, fixed.